test/fuse/README.md

```diff
@@ -101,3 +149,43 @@ SyscallFailsWithErrno(...)
 
 Please refer to [test/syscalls/README.md](../syscalls/README.md) for further
 details.
+
+## Writing a new FuseTestCmd
+
+A `FuseTestCmd` is a control protocol used in the communication between the
+testing thread and the FUSE server. The command is sent from the testing thread
+to the FUSE server to set up, control, or inspect the behavior of the FUSE
+server in response to a sequence of FUSE requests.
+
+The lifecycle of a command contains following steps:
+
+1. The testing thread sends a `FuseTestCmd` via socket and waits for completion.
+2. The FUSE server receives the command and does corresponding action.
+3. (Optional) The testing thread reads data from socket.
+4. The FUSE server sends a success indicator via socket after processing.
+5. The testing thread gets the success signal and continues testing.
+
+The success indicator is important because we don't want to begin filesystem
+operation if the fixture has not been completely set up. Also, for simplicity's
+sake, we don't use lock in memory accessing in the FUSE server. Because we
+should specify each FUSE response in a sequential manner, there is no point of
```

Comment:
Sorry I cannot help with this paragraph since I cannot understand what it's trying to say.